### PR TITLE
Feature/header styles

### PR DESCRIPTION
**Pull request by zsteiner**
New Header
New Search

### CODE REVIEW

File: src/assets/toolkit/styles/partials/_guided-filter.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .hpe-guided-filter {
   margin-bottom: 3rem;
   background: $brand url("/assets/toolkit/images/guided-background.png")
-  no-repeat;
+    no-repeat;
```
Comment (ashblue) [inline src/assets/toolkit/styles/partials/_guided-filter.scss:4]: REVENGE OF THE TABS!!! Looks like formatting bonked out on an old tab here.

File: src/assets/toolkit/styles/partials/_header.scss [hunk 2]
```diff
@@ -247,3 +272,16 @@ $nav-padding: 1rem;
     }
   }
 }
+
+.hpe-contact {
+  font-weight: 600;
+  text-transform: uppercase;
+  background: url("data:image/svg+xml;utf8, %3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%2282%22%20height%3D%2240%22%3E%3Cpath%20stroke%3D%22%2301A982%22%20stroke-width%3D%222%22%20d%3D%22M1%201v26h23v10.583L34.568%2027H81V1H1z%22%20fill%3D%22none%22%20fill-rule%3D%22evenodd%22%2F%3E%3C%2Fsvg%3E")
```
Comment (ashblue) [inline src/assets/toolkit/styles/partials/_header.scss:279]: That is one fugely mother looking line, but not much we can do about it.

predator

**Review by ashblue — APPROVED** [review]
Minor comments. Good to merge

File: src/assets/toolkit/styles/partials/_guided-filter.scss (hunk 1 above)
Comment (zsteiner) [reply src/assets/toolkit/styles/partials/_guided-filter.scss:4]: I think that's actually two spaces. I hope it is! The linter/formatter was not indenting long lines in CSS which I find really hard to read. I made that tweak to the stylelint config.

**Pull request merged**